packages/circuit-ui/components/AutocompleteInput/AutocompleteInput.mdx

```diff
@@ -0,0 +1,62 @@
+import { Meta, Status, Props, Story } from '../../../../.storybook/components';
+import * as Stories from './AutocompleteInput.stories';
+
+<Meta of={Stories} />
+
+# AutocompleteInput
+<Status variant="stable" />
```

Comment:
I would suggest initially marking the component as experimental so we have the option of making changes in minor releases.